cli/cmd/plugin/unmanaged-cluster/cluster/minikube.go

```diff
@@ -0,0 +1,122 @@
+// Copyright 2022 VMware Tanzu Community Edition contributors. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package cluster
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+	"path/filepath"
+
+	"github.com/vmware-tanzu/community-edition/cli/cmd/plugin/unmanaged-cluster/config"
+)
+
+var minikubeBinName = "minikube"
+
+type MinikubeClusterManager struct{}
+
+// Create
+func (mkcm MinikubeClusterManager) Create(c *config.UnmanagedClusterConfig) (*KubernetesCluster, error) {
+	if !mkcm.isMinikubeRunning() {
+		// TODO: Is minikube it the current context? If not, attempt to switch to it
+
+		args := []string{
+			"start",
```

Comment:
let's respect our generic nodeCount-related configurations here

Reply:
I suppose there's actually a lot of config we should respect here, like portforwarding.

Just a reminder, I'm sure you were eventually on it :smile: